src/index.ts

```diff
@@ -131,14 +132,19 @@ async function loadConfigFile<T>(filepath: string, source: LoadConfigSource<T>):
         config = await parser(filepath)
       }
       else if (parser === 'require') {
-        config = await jiti(filepath, {
-          interopDefault: true,
-          cache: false,
-          v8cache: false,
-          esmResolve: true,
-          // FIXME: https://github.com/unjs/jiti/pull/141
-          requireCache: false,
-        })(bundleFilepath)
+        if (process.env.npm_execpath.includes('bun')) {
```

Comment:
```suggestion
        if (process.versions.bun) {
```
https://github.com/vitejs/vite/pull/13901